contracts/RMRK/base/IRMRKBaseStorage.sol

```diff
@@ -57,6 +57,24 @@ interface IRMRKBaseStorage is IERC165 {
         string metadataURI; //n bytes 32+
     }
 
+    //TODO: Doc this struct, put JSON intake format in comments here
```

Comment:
Just FYI this is done in the examples documentation. I'll add it once that is merged